File: RMS-BackOfficePortal/src/app/core/models/otc-master-bal.ts
export interface OTCMasterBal{
    branch_code: string;
    bal_date: Date;

    //Validation
    result: number;

    //Listing
    daily_bal_status: string;
    no_of_counters: number;
    user_id: string;
    check_in: string;
    total_amt: number;
    total: number;

}

File: RMS/rms-service/src/main/java/com/maven/rms/models/OTCMasterBal.java
package com.maven.rms.models;

import java.math.BigDecimal;
import java.sql.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OTCMasterBal {
    //Dropdown
    private String branch_code;
    private Date bal_date;

    //Validation
    private Integer result;

    //Listing
    private String daily_bal_status;
    private String user_id;
    private Integer no_of_counters;
    private String check_in;
    private BigDecimal total_amt;
    private Integer total;
}